### PR TITLE
Do not save case attachments if feature not enabled

### PR DESCRIPTION
**Pull request by millerdev**
@mkangia @snopoke

### CODE REVIEW

File: corehq/form_processor/backends/sql/update_strategy.py [hunk 1]
```diff
@@ -222,6 +226,11 @@ def _apply_attachments_action(self, attachment_action, xform):
                     self.case.track_delete(current_attachments[name])
             elif att.attachment_src:
                 form_attachment = xform.get_attachment_meta(att.attachment_src)
+                if form_attachment is None:
+                    # Probably an improperly configured form. We need a way to
+                    # convey errors like this to domain admins.
```
Comment (proteusvacuum) [inline corehq/form_processor/backends/sql/update_strategy.py]: Nit: Not sure this is a very useful place for this comment, sounds more like a backlog item if you think someone actually needs to do this.

**Review by proteusvacuum — APPROVED** [review]

File: corehq/form_processor/backends/sql/update_strategy.py (hunk 1 above)
Comment (millerdev) [reply corehq/form_processor/backends/sql/update_strategy.py]: Fair point. In addition to being a feature suggestion, this comment is also informative that errors occurring here may be user error and possibly not directly actionable by HQ devs.

**Review by mkangia — COMMENTED** [review]
not sure what that change does. Can you elaborate a bit on why we are doing this?

**Comment by millerdev** [comment]
This change makes the form processor ignore case attachments if the feature flag that is supposed to gate their use is not enabled. It was motivated by a [bug report](https://dimagi-dev.atlassian.net/browse/HI-269).

**Review by snopoke — APPROVED** [review]

**Pull request merged**